lib/app-layout/index.js

```diff
@@ -1,19 +1,35 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, cloneElement } from 'react';
 import PropTypes from 'prop-types';
+import RevisionSelector from '../revision-selector';
 
-export const AppLayout = ({ searchBar, noteList, noteEditor }) => {
+export const AppLayout = ({
+  note,
+  revisions,
+  onUpdateContent,
+  searchBar,
+  noteList,
+  noteEditor,
+}) => {
   return (
     <Fragment>
       <div className="app-layout-source-list theme-color-bg theme-color-fg">
         {searchBar}
         {noteList}
       </div>
-      {noteEditor}
+      <RevisionSelector
+        note={note}
```

Comment:
I think you can simply pass `note` and `onUpdateContent` here since they have the same name.